integration-tests/ws-security-policy/src/main/resources/saml1-policy.xml

```diff
@@ -41,7 +19,7 @@
                     </sp:InitiatorToken>
                     <sp:RecipientToken>
                         <wsp:Policy>
-                            <sp:X509Token sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/Always">
+                            <sp:X509Token sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/AlwaysToInitiator">
                                 <wsp:Policy>
                                     <sp:WssX509V3Token10/>
                                     <sp:RequireEmbeddedTokenReference/>
```

Comment:
The policy xml will be used both at client and server side. And RecipientToken only sign the response message from server to client, so we should be more specific that this token is included only for the response message from server to client